.circleci/config.yml

```diff
@@ -0,0 +1,515 @@
+version: 2.1
+
+references:
+  ci_support_image: &ci_support_image
+                       broadinstitute/study-server-build:2020-04-10G
+  root_path: &root_path
+      /home/circleci
+  repo_path: &repo_path
+              /home/circleci/repo
+  m2_cache_key: &m2_cache_key
+                  pom.xml1-{{ checksum "pom.xml" }}
+  m2_cache_restore_key: &m2_cache_keys
+    keys:
+      - *m2_cache_key
+      - pom.xml1-
+  m2_repo_path: &m2_repo_path
+             /home/circleci/.m2/repository
+  pepper_apis-path: &pepper_apis_path
+            /home/circleci/repo/pepper-apis
+  circleci_path: &circleci_path
+            /home/circleci/repo/.circleci
+  builds_bucket: &builds_bucket
+            ddp-build-artifacts
+  jar_build_bucket_dir: &jar_build_bucket_dir
+        /ddp-study-server-pepper-apis
+  housekeeping_test_db: &housekeeping_test_db
+      housekeepingdb
+  studyserver_test_db: &studyserver_test_db
+      studyservicedb
+
+
+executors:
+  build-executor:
+    docker:
+      - image: *ci_support_image
+  test-executor:
+    docker:
+      - image: *ci_support_image
+      # start pubsub emulator
+      - image: broadinstitute/study-server-build:pubsub-1
+      - image: circleci/mysql:5.7-ram
+        environment: &DB_VARS
+          MYSQL_ROOT_PASSWORD: rootpw
+          MYSQL_DATABASE: test_db
+          MYSQL_USER: user
+          MYSQL_PASSWORD: passw0rd
+    resource_class: medium+ #default medium not big enough for tests
+
```

Comment:
Test executor starts a MySQL and a pubsub emulator. Emulator does not have to be started up from test.